gamemodes/terrortown/gamemode/roles/shadow/cl_shadow.lua

```diff
@@ -503,14 +499,7 @@ AddHook("HUDPaint", "Shadow_HUDPaint", function()
 end)
 
 AddHook("TTTHUDInfoPaint", "Shadow_Buff_TTTHUDInfoPaint", function(cli, label_left, label_top, active_labels)
-    if not cli:IsShadow() then return end
-
-    local hide_role = false
-    if ConVarExists("ttt_hide_role") then
-        hide_role = GetConVar("ttt_hide_role"):GetBool()
-    end
-
-    if hide_role then return end
+    if not cli:IsShadow() or hide_role:GetBool() then return end
```

Comment:
Just a matter of preference
```suggestion
    if not cli:IsShadow() then return end
    if hide_role:GetBool() then return end
```